firestore/src/main/query_main.cc

```diff
@@ -237,8 +237,13 @@ core::Bound QueryInternal::ToBound(
     const std::vector<FieldValue>& field_values) const {
   const core::Query& internal_query = query_.query();
   // Use explicit order bys  because it has to match the query the user made.
+#if TARGET_OS_IOS || TARGET_OS_TV
```

Comment:
Is this just from the merge from main, or related to the fix?

Reply:
Related to the fix. The next version of the iOS SDK will change this, but because on desktop it uses the source code directly, but on iOS it uses the last released cocoapods, this needs to be done in the meantime.